_data/navigation.yml

```diff
@@ -68,6 +72,8 @@ wiki:
         url: /wiki/sensing/opencv-stereo/
       - title: Camera-IMU Calibration using kalibr
         url: /wiki/sensing/camera-imu-calibration/
+      - title: SBC Basics, Sensing, and Computer Vision
```

Comment:
"SBC" is a not a common acronym and obscures the article's intent. Recommend changing to "Single Board Computers for Computer Vision."